modules/local/create_bed.nf

```diff
@@ -0,0 +1,31 @@
+process CREATE_BED {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "conda-forge::python=3.9.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'quay.io/biocontainers/python:3.9--1' }"
```

Comment:
Since it's not a python script, I would use the same `gawk` package as in https://github.com/sanger-tol/readmapping/blob/main/modules/local/custom/remove_masking.nf#L5-L8